card/spec/lib/card/query_spec.rb

```diff
@@ -381,9 +384,12 @@
       expect(Card::Query.new(:found_by=>'Image+*type+by name').run.map(&:name).sort).to eq(Card.search(:type=>'Image').map(&:name).sort)
     end
     it "should play nicely with other properties and relationships" do
-      expect(Card::Query.new(:plus=>{:found_by=>'Simple Search'}).run.map(&:name).sort).to eq(Card::Query.new(:plus=>{:name=>'A'}).run.map(&:name).sort)
+      found_by_simple = Card::Query.new( :plus=>{:found_by=>'Simple Search'}, :return=>:name, :sort=>:name ).run
```

Comment:
Line is too long. [112/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.